05_deploy_bootstrap_vm.sh

```diff
@@ -85,16 +85,15 @@ sed -i "/<\/devices>/a <qemu:commandline>\n  <qemu:arg value='-fw_cfg'/>\n  <qem
 sudo virsh define ocp/bootstrap-vm.xml
 sudo virsh start ${CLUSTER_NAME}-bootstrap
 sleep 10
-VM_MAC=$(sudo virsh dumpxml ${CLUSTER_NAME}-bootstrap | grep "mac address" | head -n 1 | cut -d\' -f2)
-while ! sudo virsh domifaddr ${CLUSTER_NAME}-bootstrap | grep -q ${VM_MAC}; do
+
+while ! domain_net_ip ${CLUSTER_NAME}-bootstrap default; do
```

Comment:
This isn't working for me, it does `python /dev/fd/63 ostest-bootstrap default` which then fails with `No interface found for bridge default`

Should this really be looking for virbr0, or perhaps better we can retrieve it from the default virsh net XML?

```
[root@tripleodev2 ~]# virsh net-dumpxml default | grep "bridge name"
  <bridge name='virbr0' stp='on' delay='0'/>
```